packages/@coorpacademy-components/src/molecule/review-correction-popin/index.native.tsx

```diff
@@ -0,0 +1,332 @@
+      fontWeight: '600',
+      lineHeight: 24,
+      marginLeft: 12,
+      textTransform: 'uppercase',
+      wordBreak: 'break-word'
+    },
+    feedbackSection: {
+      marginVertical: theme.spacing.medium
+    },
+    labelContainer: {
+      alignSelf: 'flex-start',
+      backgroundColor: 'rgba(255, 255, 255, 0.3)',
+      borderRadius: 56,
+      paddingHorizontal: theme.spacing.tiny,
+      paddingVertical: theme.spacing.micro
+    },
+    label: {
+      color: theme.colors.white,
+      fontSize: 14,
+      fontWeight: theme.fontWeight.extraBold,
+      lineHeight: 17,
+      wordBreak: 'break-word'
+    },
+    button: {
+      alignSelf: 'stretch',
+      backgroundColor: theme.colors.white,
+      borderRadius: 7,
+      paddingHorizontal: theme.spacing.base,
+      paddingVertical: theme.spacing.small
+    },
+    buttonText: {
+      color: theme.colors.cta,
+      fontSize: 14,
+      fontWeight: theme.fontWeight.bold,
+      lineHeight: 20,
+      textAlign: 'center'
+    },
+    containerButtonKlf: {
+      width: '100%'
+    },
+    buttonKlf: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: 'rgba(255, 255, 255, 0.1)',
+      borderRadius: 7,
+      paddingHorizontal: theme.spacing.base,
+      paddingVertical: theme.spacing.small,
+      marginBottom: theme.spacing.tiny
+    },
+    buttonKlfActive: {
+      backgroundColor:
+        'linear-gradient(0deg, rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 0.25)), rgba(255, 255, 255, 0.1)'
+    },
+    buttonKlfText: {
+      color: 'white',
+      fontSize: 14,
+      fontWeight: theme.fontWeight.bold,
+      lineHeight: 20,
+      textAlign: 'center'
+    },
+    iconKey: {
+      width: 12,
+      height: 12,
+      marginRight: theme.spacing.tiny
+    },
+    containerTooltip: {
+      zIndex: 30,
+      position: 'relative',
+      shadowColor: theme.colors.black,
+      shadowOpacity: 0.3,
+      shadowOffset: {width: 0, height: 0},
+      shadowRadius: 8
+    },
+    buttonTooltip: {
+      flexDirection: 'column',
+      justifyContent: 'center',
+      backgroundColor: theme.colors.white,
+      borderRadius: 15,
+      padding: theme.spacing.small,
+      width: '112%',
+      position: 'absolute',
+      bottom: 4,
+      right: -15,
+      zIndex: 20
+    },
+    triangleTooltip: {
+      width: 0,
+      height: 0,
+      backgroundColor: 'transparent',
+      borderStyle: 'solid',
+      borderLeftWidth: 13,
+      borderRightWidth: 13,
+      borderBottomWidth: 13,
+      borderTopWidth: 13,
+      borderLeftColor: 'transparent',
+      borderTopColor: 'transparent',
+      borderRightColor: theme.colors.white,
+      borderBottomColor: theme.colors.white,
+      transform: [{rotate: '45deg'}],
+      position: 'absolute',
+      left: 110,
+      top: -20,
+      borderRadius: 4
+    }
+  });
+
+const KlfButton = ({
+  klf,
+  styleSheet
+}: {
+  klf: NonNullable<ReviewCorrectionPopinProps['klf']>;
```

Comment:
Isn't it better to modify the `prop-types.ts` file, exporting a type ReviewCorrectionPopinKLFProps, to use it here, and to add it as non mandatory on

```js
export type ReviewCorrectionPopinProps = {
  klf?: ReviewCorrectionPopinKLFProps
  ....
}
```
?


It will allow you to not use this NonNullable tag that makes your code less readable.